pkg/istio-agent/xds_proxy.go

```diff
@@ -61,11 +61,24 @@ import (
 
 const (
 	defaultClientMaxReceiveMessageSize = math.MaxInt32
-	defaultInitialConnWindowSize       = 1024 * 1024     // default gRPC InitialWindowSize
-	defaultInitialWindowSize           = 1024 * 1024     // default gRPC ConnWindowSize
-	sendTimeout                        = 5 * time.Second // default upstream send timeout.
+	defaultInitialConnWindowSize       = 1024 * 1024 // default gRPC InitialWindowSize
+	defaultInitialWindowSize           = 1024 * 1024 // default gRPC ConnWindowSize
 )
 
+var (
+	sendTimeout = 5 * time.Second // default upstream send timeout.
```

Comment:
@ramaraochavali should we also just bump up the default?

Reply:
5s seems a decent the timeout. I think let us leave it for now.

Reply:
I thought it was super high but I think what happens is the LDS push takes 5+s, so the next request (RDS, for example) is blocked. Its not 5s network latency.

But still pretty high...

Reply:
Yeah. Got it. Just proxying takes more than 5s is strange to me. Is it just because of too much data?